pennylane/io.py

```diff
@@ -410,113 +407,131 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
-def from_qasm(quantum_circuit: str, measurements=False):
-    """Loads quantum circuits from a QASM string using the converter in the
+def from_qasm(quantum_circuit: str, measurements=None):
+    """
+    Loads quantum circuits from a QASM string using the converter in the
     PennyLane-Qiskit plugin.
 
     Args:
         quantum_circuit (str): a QASM string containing a valid quantum circuit
         measurements (None | MeasurementProcess | list[MeasurementProcess]): an optional PennyLane
-            measurement or list of PennyLane measurements that overrides any terminal measurements
-            that may be present in the input circuit. If set to ``None``, existing measurements
-            in the input circuit will be used.
+            measurement or list of PennyLane measurements that overrides the terminal measurements
+            that may be present in the input circuit. Defaults to ``None``, such that existing measurements
```

Comment:
```suggestion
            that may be present in the input circuit. Defaults to ``None``, such that existing measurements
```

Reply:
I think for the args description there's no need to update this - details make more sense further down